packages/babel-plugin-react-server/src/index.js

```diff
@@ -0,0 +1,34 @@
+import loggerSpec from 'react-server-module-tagger';
+import path from 'path';
+
+module.exports = function({types: t }) {
+  return {
+    visitor: {
+      Identifier(p, state) {
+        const {node} = p;
+        const {name, type} = node;
+
+        const config = { trim: state.opts.trim };
+        const parent = path.resolve(path.join(process.cwd(), '..')) + path.sep;
+        const fp = this.file.opts.filename.replace(parent, '');
+        const file =  { path: fp };
```

Comment:
Yeah, a [nicer interface](https://github.com/redfin/react-server/pull/300#discussion_r68860295) for the module tagger would eliminate the need for this nonsense.